styles/Header.module.css

```
@@ -1,8 +1,333 @@
 
-.link {
-    color: #000000;
+.navBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    height: 75px;
+    padding: 0 50px;
+    position: sticky;
+    top: 0;
+    background-color: white;
+    box-shadow: 0px 5px 15px 0 rgba(0, 0, 0, 0.15);
+}
+
+.links {
+    display: inline-block;
+    padding: 10px 15px;
+    text-decoration: none;
+    color: black;
 }
 
 .activeLink {
-    color: #ff0000;
-}
\ No newline at end of file
+    color: #5CA2D5;
+    text-decoration: underline;
+    font-weight: bold;
+}
+
+.logo {
+    display: inline-block;
+    height: 65px;
```

So minor, but can you make this 60px instead? The image looks a bit close to the sides of the header bar right now
```suggestion
    height: 60px;
```